src/lib/openapi.ts

```diff
@@ -1,5 +1,91 @@
+import { openapi } from '@seamapi/types/connect'
```

Comment:
We should not import `@seamapi/types/connect` here. We need the blueprint to define the (subset) of the openapi schema we will parse and then `@seamapi/types/connect` has to implement it.

One issues we have had with the existing openapi schema is when stuff was missing. We can either import an Openapi schmea typed defined in a 3rd party lib, or define a stricter openapi schema here (the proposed approach).

Reply:
This test is already checking the blueprint generation for `@seamapi/types/connect` https://github.com/seamapi/blueprint/blob/main/test/seam-blueprint.test.ts

Reply:
This fixture can be manually crafted to test all edge cases / parsing in a more controlled way: https://github.com/seamapi/blueprint/blob/main/test/fixtures/types/openapi.ts

Reply:
Oh, okay. So the types repo will be solely responsible for the generation? Guess I could move my efforts there once we define what we need.

Reply:
No, this repo will generate the blueprint from the types repo. [This test](https://github.com/seamapi/blueprint/blob/main/test/seam-blueprint.test.ts) show exactly how the sdk gen or doc gen code would use this project. So for example, 

```ts
import { openapi } from '@seamapi/types/connect'
import { createBlueprint } from '@seamapi/blueprint'

const generateSdk = async () => {
  const blueprint = createBlueprint({ openapi })
  await generateAndWriteRouteClasses(blueprint.routes)
  ...more codegen
}

Reply:
Oh, okay. So it's the createBlueprint that consumes the openapi schema and creates a reusable, more accessible format? I'll try to implement it then.